### PR TITLE
test: add event checking to TestInstantiateContract2

### PR DESCRIPTION
**Pull request by da1suk8**
### TestInstantiateContract2
https://github.com/Finschia/wasmd/blob/main/docs/proto/proto-docs.md#msginstantiatecontract2

This PR adds an integration test for TestInstantiateContract2.

This test is based on https://github.com/CosmWasm/wasmd/blob/33df4aeaaa419f6aad07b94d64c0423bee0a2d21/x/wasm/keeper/msg_server_integration_test.go#L899-L976

## Checklist:

- [x] I followed the [contributing guidelines](https://github.com/Finschia/wasmd/blob/main/CONTRIBUTING.md) and [code of conduct](https://github.com/Finschia/wasmd/blob/main/CODE_OF_CONDUCT.md).
- [x] I have added a relevant changelog to `CHANGELOG.md`
- [ ] I have added tests to cover my changes.
- [ ] I have updated the documentation accordingly.
- [ ] I have updated API documentation `client/docs/swagger-ui/swagger.yaml`

### CODE REVIEW

**Review by kokeshiM0chi — DISMISSED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,6 +12,8 @@
 * [\#64](https://github.com/Finschia/wasmd/pull/64) test: add test cases to confirm output for PinnedCodes
 * [\#70](https://github.com/Finschia/wasmd/pull/70) add event checking to TestInstantiateContract
 * [\#73](https://github.com/Finschia/wasmd/pull/73) test: add the check for expPaginationTotal
+* [\#74](https://github.com/Finschia/wasmd/pull/74) add event checking to TestInstantiateContract2
+
```
Comment (170210) [inline CHANGELOG.md:16]:
```suggestion
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -12,6 +12,8 @@
...
```
Comment (170210) [inline CHANGELOG.md:17]:
```suggestion

```

File: x/wasm/keeper/msg_server_integration_test.go [hunk 3]
```diff
@@ -141,3 +141,87 @@ func TestInstantiateContract(t *testing.T) {
...
+			assert.Equal(t, "message", events[0].Type)
+			assert.Equal(t, 2, len(events[0].Attributes))
+			assert.Equal(t, "module", string(events[0].Attributes[0].Key))
+			assert.Equal(t, "wasm", string(events[0].Attributes[0].Value))
+			assert.Equal(t, "sender", string(events[0].Attributes[1].Key))
+			assert.Equal(t, myAddress.String(), string(events[0].Attributes[1].Value))
+			assert.Equal(t, "instantiate", events[1].Type)
+			assert.Equal(t, 2, len(events[1].Attributes))
+			assert.Equal(t, "_contract_address", string(events[1].Attributes[0].Key))
+			assert.Contains(t, string(rsp.Data), string(events[1].Attributes[0].Value))
```
Comment (170210) [inline x/wasm/keeper/msg_server_integration_test.go:220]: I think it is better to use equal

Comment (da1suk8) [reply x/wasm/keeper/msg_server_integration_test.go]: Fixed thank you!

**Review by 170210 — APPROVED** [review]

**Review by kokeshiM0chi — APPROVED** [review]

**Pull request merged**